routes/claim.py

```diff
@@ -7,6 +7,7 @@ async def claim_cell(request: web.Request):
     body = loads(await request.text())
     room, player = GameServer.instance().find_player(body['login'])
     if room is None: return web.Response(status=404)
+    if player.secret != body['client_secret']: return web.Response(status=405)
```

Comment:
The HTTP code 405 suggests "Method not allowed" which I'm not sure is the most suitable here. That doesn't really matter though as we're not exposing this in a public API.